src/Blorc.OpenIdConnect.Tests/Extensions/ObjectExtensionsFacts.cs

```diff
@@ -112,7 +176,7 @@ public void Collects_Claims_From_Complex_Type()
 
                 var claims = complexType.AsClaims().ToList();
 
-                Assert.That(claims.Count, Is.EqualTo(18));
+                Assert.That(claims.Count, Is.EqualTo(30));
```

Comment:
I want to introduce something else here. We have having really good experiences with VerifyTests. This will give you a visual representation of the expected and actual results, making it easier to compare and approve.

Should be a "one-liner", something like:

`await Verifier.Verify(claims);`

If you don't want to investigate this, we can do it ourselves later.